test/new_project.sh

```diff
@@ -1,10 +1,22 @@
 #!/bin/sh
 set -ex
-rm  -rf tmp-project
+# remove temporary blank project
+rm  -rf test/tmp-project
+cd test/
+# test generating new project in cwd
 mkdir tmp-project
 cd tmp-project
-../bin/near new_project
+../../bin/near new_project
 npm install
 npm uninstall near-shell
-npm install ../
+npm install ../../
 NODE_ENV=development npm run test
+cd ..
+rm  -rf tmp-project
+# test generating new project in new dir
```

Comment:
full test cycle is pretty slow, I think it's enough to test option with explicit directory full cycle and just check that both options write out `package.json`

Reply:
Can run tests in the first cycle and will just check the package.json moves in the second.